backend/actions/transaction_list.js

```diff
@@ -91,25 +147,75 @@ app.get('/transactions', async (req, res) => {
 
       getBlockchainInfo().then( async (bestBlockHeight) => {
 
-        let count = 0, transList= [], overheadTxCount = 0;
-        const memTxList = await getMemTx();
-        if(( memTxList.length) > (perPage*(page-1))){
-
-          let j = (perPage*(page-1));
-          while(j < memTxList.length){ 
-            let amount = 0;
-            memTxList[j].vout.forEach( (vout) => {amount += vout.value})
-            memTxList[j].amount = amount;
-            memTxList[j].confirmations = 0;
-            transList.push(memTxList[j]);
-            j++;
-            count++;
-            if(count == perPage){
-              break;
+        //let cacheBestBlockHeight = cache.getKey(`bestBlockHeight`);
+
+        createCache().then( async () => {
+
+          const cache = flatCache.load('cacheId');
+          
+          let count = 0, transList= [];
+          const memTxList = await getMemTx();
+          if(( memTxList.length) > (perPage*(page-1))){
+  
+            let j = (perPage*(page-1));
+            while(j < memTxList.length){ 
+              let amount = 0;
+              memTxList[j].vout.forEach( (vout) => {amount += vout.value})
+              memTxList[j].amount = amount;
+              memTxList[j].confirmations = 0;
+              transList.push(memTxList[j]);
+              j++;
+              count++;
+              if(count == perPage){
+                break;
+              }
             }
           }
-        }
+                    
+          if(cache.getKey(`bestBlockHeight`) === bestBlockHeight){
+            const transactionCount = await getTxCount();
```

Comment:
How about using the variable txCount received at line 146, instead of getTxCount() again?

Reply:
Okay Sir, will change it. Thank You